addons/l10n_be/models/template_be.py

```diff
@@ -92,6 +101,8 @@ def _get_be_reconcile_model(self):
                         'label': 'Virements internes',
                     }),
                 ],
-                'name@nl': 'Interne overboekingen',
+                'name@fr': 'Virements internes',
+                'name@nl': 'Overschrijvingskosten',
```

Comment:
```suggestion
                'name@nl': 'Interne overboekingen',
```
(I asked a Dutch speaker to be sure :smile: )